examples/graphQL/toolpad/resources/function.ts

```diff
@@ -0,0 +1,46 @@
+import { request } from 'graphql-request';
+
+export async function queryStargazers(owner: string, repository: string) {
+  if (!process.env.GITHUB_TOKEN) {
+    throw new Error(`Env variable GITHUB_TOKEN not configured`);
+  }
+
+  const endpoint = 'https://api.github.com/graphql';
+  const token = process.env.GITHUB_TOKEN;
+
+  const query = `
```

Comment:
We can use https://www.npmjs.com/package/graphql-tag for these template strings.

Reply:
I added this and expected syntax highlighting, but nothing happened.